src/signalo/core/migrations/0001_initial.py

```diff
@@ -29,7 +46,7 @@ class Migration(migrations.Migration):
                 (
                     "geom",
                     django.contrib.gis.db.models.fields.PointField(
-                        srid=4326, verbose_name="Geometry"
+                        srid=2056, verbose_name="Geometry"
```

Comment:
this needs to be set from the settings

Reply:
Yes but this is just a migration after all, written from raw values...